features/cart_page.feature

```diff
@@ -14,7 +14,8 @@ Feature: As a Customer
       | Salad     | Leafy           | 200        |
 
   Scenario: Adding dish to cart
-    Given I am on the dish page for "Pizza"
+    Given I am not logged in
```

Comment:
My objection to you using this step in this particular scenario is that we don't log in a user in the background. So there is no user logged in no matter what, right?